cybershell/scoring.py

```
@@ -0,0 +1,482 @@
+from typing import Dict, Any, List, Optional
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
```

Ruff (0.12.2)

`dataclasses.dataclass` imported but unused

Remove unused import: `dataclasses.dataclass`

(F401)